src/Compilers/CSharp/Portable/Syntax/SyntaxExtensions.cs

```diff
@@ -384,7 +384,7 @@ private static bool IsInContextWhichNeedsDynamicAttribute(CSharpSyntaxNode node)
         internal static bool IsOutDeclaration(this DeclarationExpressionSyntax p)
         {
             return p.Parent?.Kind() == SyntaxKind.Argument
```

Comment:
The analyzer is possible missing this.

Reply:
Seems likely, yes

Reply:
Filed https://github.com/dotnet/roslyn-analyzers/issues/4946